website/emp-informatics-working-group.md

```diff
@@ -0,0 +1,28 @@
+The EMP Informatics Working Group is charged with the implementation of
+data handling, analysis and publication following the generation of
+sequencing data. Their charge includes the generation and maintenance of
+the metadata repository The Global Environmental Sample Database (GESD).
+**Chief Data Analyst**: Greg Caporaso (Northern Arizona University, USA)
+**Data Analysis Team:** Sean Gibbons - Argonne National Laboratory,
+University of Chicago, USA Rob Knight - University of Colorado at
+Boulder, USA Christopher Quince - University of Glasgow, UK Alice
+McHardy - MPI Dusseldorf, Germany Rick Stevens, Argonne National
+Laboratory, USA **Global Environmental Sample Database Coordinator**:
+Doug Wendel - University of Colorado at Boulder, USA **GESD Team:**
+Sarah Owens - Argonne National Laboratory, USA
+<div>
+
+Gail Lesley Ackermann - University of Colorado at Boulder, USA
+
+</div>
+
+<div>
+
+Giancarlo Galindo - Lawrence Berkley National Laboratory, USA
+
+</div>
+
+**Data Publication (MGRAST Team):** Folker Meyer - Argonne National
```

Comment:
I'm not sure...
Maybe we could send a link around and get people to check. Here is the version I'm working on:
https://github.com/colinbrislawn/emp/blob/patch-1/website/emp-informatics-working-group.md